ccc/tests/test_parameters.py

```diff
@@ -58,7 +58,7 @@ def test_update_bad_revision1():
     first_line = spec.errors['profit_rate'][0]
     print(first_line)
     expected_first_line =\
-        'profit_rate 1.2 must be less than 1.0 for labels year=2019.'
+        'profit_rate[year=2019] 1.2 > max 1.0 '
```

Comment:
@hdoupe I like the new error message format - but notice the white space at the end of the string.  Any reason for that?  If not, you might want to clean that up.

Reply:
Thanks @jdebacker. Hat-tip to Tax-Calculator for the basic lay-out of the messages.

Nope, I'll fix the extra white space.